lib/autokey/scripting/engine.py

```diff
@@ -365,6 +371,29 @@ def run_script_from_macro(self, args):
         except Exception as e:
             self.set_return_value("{ERROR: %s}" % str(e))
 
+    def get_script_arguments(self):
+        """
+        Get the arguments supplied to the current script via the scripting api
+
+        Usage: C{engine.get_script_arguments()}
+
+        @return: the arguments
+        @rtype: C{list(str())}
```

Comment:
This is of type `List[Any]`. The user can and should be able to pass anything.

Reply:
Thanks, I misunderstood how the doc worked.